src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23488.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Tests.Issues
+{
+    public class Issue23488 : _IssuesUITest
+    {
+		public Issue23488(TestDevice device): base(device)
+		{
+		}
+
+		public override string Issue => "Span text-decoration is incorrect whereas the Label behaves properly";
+
+		[Test]
+		[Category(UITestCategories.Label)]
+		public async Task LabelHyperlinkUnderlineColor()
+		{
+			await Task.Delay(500);
```

Comment:
Add an automation ID to the `Label` here https://github.com/dotnet/maui/pull/24063/files#diff-b1f8b9006e0c54ae7bc49f411b3594a69604ede70d85508200debae016192994R8 and then use `App.WaitForElement` on that AutomationId

Reply:
Added automation ID to the label.